### PR TITLE
SEOImages: User Story 1463880, Part 34

### PR DESCRIPTION
**Pull request by damabe**
## Summary

Updating image file names and references to improve SEO.

Fixes #1463880

Contributes to #10839

### CODE REVIEW

**Comment by damabe** [comment]
@mairaw I've updated the identified issues (file names without hyphens) in these files. The index.md file in this PR has a good deal of links that are not close to the 20 character minimum. Also, they don't have alt text for any of them so they are causing a lot of errors in the CATS report: https://contentqacats.azurewebsites.net/CheckResults/2019032100062/Report/.

Please let me know if you have a suggestion for the short links. For example, the visual basic icon/logo is named vb.svg. I updated it to visual-basic.svg but that still is not close to the minimum.

File: docs/framework/windows-workflow-foundation/how-to-create-a-flowchart-workflow.md [hunk 1]
```diff
@@ -136,7 +136,7 @@ Workflows can be constructed from built-in activities as well as from custom act
   
      The following example illustrates the completed workflow.  
   
-     ![Completed Windows Workflow Foundation](./media/gettingstartedtutorialcompletedflowchart.PNG "GettingStartedTutorialCompletedFlowchart")  
+     ![Diagram that shows a completed Windows Workflow flowchart.](./media/how-to-create-a-flowchart-workflow/completed-windows-workflow-flowchart.png)
```
Comment (mairaw) [inline docs/framework/windows-workflow-foundation/how-to-create-a-flowchart-workflow.md]:
```suggestion
     ![Diagram that shows a completed Windows Workflow Foundation flowchart.](./media/how-to-create-a-flowchart-workflow/completed-windows-workflow-flowchart.png)  
```

File: docs/framework/winforms/advanced/how-to-tile-a-shape-with-an-image.md [hunk 2]
```diff
@@ -20,11 +20,11 @@ Just as tiles can be placed next to each other to cover a floor, rectangular ima
   
 -   This example uses the following 75×75 image to tile a 200×200 rectangle.  
   
- ![Tile 1](./media/tile1.gif "tile1")  
+ ![Screenshot that shows a tile image.](./media/how-to-tile-a-shape-with-an-image/rectangle-tile-200x200.gif)
```
Comment (mairaw) [inline docs/framework/winforms/advanced/how-to-tile-a-shape-with-an-image.md]:
```suggestion
 ![The tile image that shows a red house and a tree.](./media/how-to-tile-a-shape-with-an-image/rectangle-tile-200x200.gif)  
```

File: docs/framework/winforms/advanced/how-to-tile-a-shape-with-an-image.md [hunk 3]
```diff
@@ -20,11 +20,11 @@ Just as tiles can be placed next to each other to cover a floor, rectangular ima
   
 -   This example uses the following 75×75 image to tile a 200×200 rectangle.  
   
- ![Tile 1](./media/tile1.gif "tile1")  
+ ![Screenshot that shows a tile image.](./media/how-to-tile-a-shape-with-an-image/rectangle-tile-200x200.gif)  
   
 -   The following illustration shows how the rectangle is tiled with the image. Note that all tiles have the same orientation; there is no flipping.  
   
- ![Tile 2](./media/tile2.gif "tile2")  
+ ![Screenshot of a rectangle tiled with an image.](./media/how-to-tile-a-shape-with-an-image/rectangle-tiled-image-no-flip.gif)
```
Comment (mairaw) [inline docs/framework/winforms/advanced/how-to-tile-a-shape-with-an-image.md]:
```suggestion
 ![A rectangle tiled with the image using the same orientation for all tiles.](./media/how-to-tile-a-shape-with-an-image/rectangle-tiled-image-no-flip.gif)  
```

File: docs/framework/winforms/advanced/how-to-tile-a-shape-with-an-image.md [hunk 4]
```diff
@@ -33,7 +33,7 @@ Just as tiles can be placed next to each other to cover a floor, rectangular ima
   
 -   This example uses the same 75×75 image to fill a 200×200 rectangle. The wrap mode is set to flip the image horizontally. The following illustration shows how the rectangle is tiled with the image. Note that as you move from one tile to the next in a given row, the image is flipped horizontally.  
   
- ![Tile 3](./media/tile3.gif "tile3")  
+ ![Screenshot that shows a tile flipped horizontally.](./media/how-to-tile-a-shape-with-an-image/rectangle-tiled-image-horizontal-flip.gif)
```
Comment (mairaw) [inline docs/framework/winforms/advanced/how-to-tile-a-shape-with-an-image.md]:
```suggestion
 ![A rectangle tiled with the image flipped horizontally.](./media/how-to-tile-a-shape-with-an-image/rectangle-tiled-image-horizontal-flip.gif)  
```

File: docs/framework/winforms/advanced/how-to-tile-a-shape-with-an-image.md [hunk 5]
```diff
@@ -49,7 +49,7 @@ Just as tiles can be placed next to each other to cover a floor, rectangular ima
   
 -   This example uses the same 75×75 image to tile a 200×200 rectangle. The wrap mode is set to flip the image both horizontally and vertically. The following illustration shows how the rectangle is tiled by the image. Note that as you move from one tile to the next in a given row, the image is flipped horizontally, and as you move from one tile to the next in a given column, the image is flipped vertically.  
   
- ![Tile 5](./media/tile5.gif "tile5")  
+ ![Screenshot that shows a tile flipped horizontally and vertically.](./media/how-to-tile-a-shape-with-an-image/rectangle-tiled-image-horizontal-vertical-flip.gif)
```
Comment (mairaw) [inline docs/framework/winforms/advanced/how-to-tile-a-shape-with-an-image.md]:
```suggestion
 ![A rectangle tiled with the image flipped horizontally and vertically.](./media/how-to-tile-a-shape-with-an-image/rectangle-tiled-image-horizontal-vertical-flip.gif)  
```

File: docs/framework/winforms/advanced/how-to-use-a-color-matrix-to-transform-a-single-color.md [hunk 6]
```diff
@@ -41,7 +41,7 @@ ms.assetid: 44df4556-a433-49c0-ac0f-9a12063a5860
   
  The following illustration shows the original image on the left and the transformed image on the right.  
   
- ![Colors](./media/colortrans1.png "colortrans1")  
+ ![Screenshot that shows the color transformation of an image.](./media/how-to-use-a-color-matrix-to-transform-a-single-color/color-transformation.png)
```
Comment (mairaw) [inline docs/framework/winforms/advanced/how-to-use-a-color-matrix-to-transform-a-single-color.md]:
```suggestion
 ![A purple square on the left and a fuchsia square on the right.](./media/how-to-use-a-color-matrix-to-transform-a-single-color/color-transformation.png)  
```

**Review by mairaw — CHANGES_REQUESTED** [review]
Thanks @damabe. Changes look good but left some feedback. Sorry for the delay!

**Comment by damabe** [comment]
@mairaw Thanks for the feedback. The updates have been made.

**Review by mairaw — APPROVED** [review]
Thank you @damabe!

**Pull request merged**